manifests/0000_51_machine-approver-02-clusterrole.yaml

```diff
@@ -18,3 +18,25 @@ rules:
   - certificatesigningrequests/approval
   verbs:
   - update
+- apiGroups:
```

Comment:
These changes do not look correct to me.

Even if we need these permissions, we probably do not need them at the cluster scope.